transpose.py

```diff
@@ -136,15 +110,27 @@ def render(table, params, *, input_columns):
         # Convert everything to text before converting. (All values must have
         # the same type.)
         to_convert = [c for c in table.columns if input_columns[c].type != "text"]
-        colnames_auto_converted_to_text.extend(to_convert)
-        if len(to_convert) == 1:
-            start = f'Column "{to_convert[0]}" was'
-        else:
-            colnames = ", ".join(f'"{c}"' for c in to_convert)
-            start = f"Columns {colnames} were"
+        cols_str = ", ".join(f'"{c}"' for c in to_convert)
```

Comment:
Let's not do this message formatting. It's not multilingual (and it doesn't even read properly in English.) Instead, please pass arguments `{n_columns}`, `{first_colname}`.